remote_monitoring/remote_monitoring/metrics_export.py

```diff
@@ -32,14 +32,13 @@ def get_data(qibocal_output_folder: Path) -> QpuData:
     acquisition_time = report_meta.start_time
     qpu_data = collections.defaultdict(dict)
 
-    for task_id, result in out.history.items():
-        task_id = task_id.id
-        metric = METRICS[task_id]
-        metric_values = getattr(result.results, metric)
-        for qubit_id, qubit_metric in metric_values.items():
-            if isinstance(qubit_metric, list):
-                qubit_metric = qubit_metric[0]
-            qpu_data[qubit_id][metric] = qubit_metric
+    calibration = Calibration.model_validate_json(
+        (qibocal_output_folder / "new_platform" / "calibration.json").read_text()
+    )
+    for qubit_id, qubit_calibration in calibration.single_qubits.items():
+        qpu_data[qubit_id]["t1"] = qubit_calibration.t1[0]
+        qpu_data[qubit_id]["t2"] = qubit_calibration.t2[0]
+        qpu_data[qubit_id]["assignment_fidelity"] = qubit_calibration.readout.fidelity
```

Comment:
```suggestion
        qpu_data[qubit_id]["assignment_fidelity"] = qubit_calibration.readout.assignment_fidelity
```
If you want the assignment fidelity otherwise keep the other one if you want the readout fidelity.
We have a property in qibocal for computing the assignment fidelity from the readout fidelity

https://github.com/qiboteam/qibocal/blob/09b7b2b2efbaebd385fcd4a2364d7f6275e5116d/src/qibocal/calibration/calibration.py#L119-L122

Reply:
Since the sql attribute name is "assignment_fidelity", we can use `assignment_fidelity` for now.

Reply:
Generally people are using the assignment fidelity because it is higher compared to the readout fidelity. I think it is the correct choice.